chat/message/src/MessageBanner/MessageBanner.types.ts

```diff
@@ -0,0 +1,17 @@
+import { Variant } from '@leafygreen-ui/banner';
+import { DarkModeProps, HTMLElementProps } from '@leafygreen-ui/lib';
+
+export interface MessageBannerProps
+  extends HTMLElementProps<'div', never>,
+    DarkModeProps {
+  /**
+   * Determines the color and glyph of the MessageBanner.
+   * @default Variant.Info
+   */
+  variant?: Variant;
+
+  /**
+   * The content inside of the MessageBanner.
+   */
+  children: React.ReactNode | Array<React.ReactNode>;
```

Comment:
`ReactNode` includes `Iterable<ReactNode>`, so we shouldn't need the extra `Array` definition